app/src/main/java/com/example/next_show/adapters/ShowAdapter.java

```diff
@@ -109,6 +122,70 @@ private void setUpView() {
             tvShowTitle = itemView.findViewById(R.id.tvShowTitle);
             tvShowBody = itemView.findViewById(R.id.tvShowBody);
             ivCardImage = itemView.findViewById(R.id.ivCardImage);
+
+            // swipe layout set up
+            swipe_layout = itemView.findViewById(R.id.swipe_layout);
+            ivDismiss = itemView.findViewById(R.id.ivDismiss);
+
+            // set listener for swiping
+            swipe_layout.setOnActionsListener(new SwipeLayout.SwipeActionsListener() {
```

Comment:
Perhaps the issue was that you didn't set the click listener on the swipe layout?  And instead, it's just on the card view?